cfgov/data_research/tests/management/commands/test_conference_export.py

```diff
@@ -1,8 +1,13 @@
+from __future__ import unicode_literals
```

Comment:
If you don't feel strongly, that might be best, if only so that if someone checks the commit log on those files they don't see this PR.